ElementX/Sources/Screens/RoomScreen/RoomScreenModels.swift

```diff
@@ -68,14 +68,12 @@ enum RoomScreenViewPollAction {
     case edit(pollStartID: String, poll: Poll)
 }
 
-enum RoomScreenViewAudioAction {
+enum RoomScreenViewAudioPlayerAction {
```

Comment:
Does the `View` make any senes in here?
```suggestion
enum RoomScreenAudioPlayerAction {
```